tidy3d/web/api/container.py

```diff
@@ -236,7 +311,33 @@ def estimate_cost(self, verbose: bool = True) -> float:
 
 
 class BatchData(Tidy3dBaseModel):
-    """Holds a collection of data objects returned by :class:`.Batch`."""
+    """
+    Holds a collection of :class:`.SimulationData` returned by :class:`Batch`.
+
+    Notes
+    -----
+
+        When the batch is completed, the output is not a :class:`.SimulationData` but rather a :class:`BatchData`. The
+        data within this :class:`BatchData` object can either be indexed directly ``batch_results[task_name]`` or can be looped
+        through ``batch_results.items()`` to get the :class:`.SimulationData` for each task.
```

Comment:
I would say "to get the `task_name` and :class:`.SimulationData` for each task, similar to iterating over a dictionary".